### PR TITLE
DA-001: fix incorrect script aliases

### PR DESCRIPTION
**Pull request by Iriakuma**
Removed single quotes where it was not required in the script

### CODE REVIEW

File: script.sql [hunk 1]
```diff
@@ -104,15 +104,15 @@ select
 	order_amount,
     ex_rate,
     currency,
-    round((order_amount * ex_rate),1)  as 'lifetime_purchase'
```
Comment (IanKoegelenberg) [inline script.sql:107]: @Iriakuma this looks good, however please remove the duplicate whitespace o 107 and 108.

**Review by IanKoegelenberg — CHANGES_REQUESTED** [review]
Please fix as per review comments.

File: script.sql (hunk 1 above)
Comment (Iriakuma) [reply script.sql:107]: Done

**Review by IanKoegelenberg — APPROVED** [review]
Good job!

**Pull request merged**